src/LaravelDebugbar.php

```diff
@@ -591,6 +591,12 @@ public function modifyResponse(Request $request, Response $response)
             } catch (\Exception $e) {
                 $app['log']->error('Debugbar exception: ' . $e->getMessage());
             }
+        } elseif ($app['config']->get('debugbar.inject', true) && $app['config']->get('debugbar.csp-compatible', true)) {
```

Comment:
You are right. I overlooked this default value. I will change it next Monday and open new pull request.